src/themes/tufte/styles.css

```
@@ -0,0 +1,76 @@
+@import '../../extensions/cite-apa/styles.css';
+@import '../../extensions/code/styles.css';
+@import '../../extensions/math/styles.css';
+@import '../../extensions/pages/styles.css';
+@import '../../extensions/person/styles.css';
+
+@import '../../fonts/etBook/etBook.css';
+@import 'https://unpkg.com/tufte-css@1.7.2/tufte.min.css';
+
+:--root {
+  --font-size-80: 80%;
```

An optional, but nice to have, set of variables to set here would be for the font-family-x ones so that they get applied to the web component elements such as buttons and tooltips.

```suggestion
  --font-family-display:  et-book,Palatino,Palatino Linotype,Palatino LT STD,Book Antiqua,Georgia,serif;
  --font-family-body: et-book,Palatino,Palatino Linotype,Palatino LT STD,Book Antiqua,Georgia,serif;
  --font-size-80: 80%;
```